packages/tasit-sdk/src/Decentraland.forkedRopsten.test.js

```diff
@@ -0,0 +1,221 @@
+import { expect, assert } from "chai";
+
+import { Account, Action } from "./TasitSdk";
+const { ERC20, ERC721, Marketplace, ConfigLoader } = Action;
+const { Mana } = ERC20;
+const { Estate, Land } = ERC721;
+const { Decentraland: DecentralandMarketplace } = Marketplace;
+import config from "./config/default";
+
+import { ropsten as ropstenAddresses } from "../../tasit-contracts/decentraland/addresses";
+const {
+  MarketplaceProxy: MARKETPLACE_ADDRESS,
+  LANDProxy: LAND_ADDRESS,
+  MANAToken: MANA_ADDRESS,
+  EstateProxy: ESTATE_ADDRESS,
+} = ropstenAddresses;
+
+import {
+  createSnapshot,
+  revertFromSnapshot,
+  confirmBalances,
+  gasParams,
+  setupWallets,
+  addressesAreEqual,
+  bigNumberify,
+  etherFaucet,
+  ropstenManaFaucet,
+  constants,
+  ProviderFactory,
+  DecentralandUtils,
+} from "./testHelpers/helpers";
+
+const { ONE, TEN, BILLION } = constants;
+
+const ROPSTEN_NETWORK_ID = 3;
+
+describe("Decentraland tasit app test cases (ropsten)", () => {
+  let ownerWallet;
+  let ephemeralWallet;
+  let manaContract;
+  let landContract;
+  let estateContract;
+  let marketplaceContract;
+  let landForSale;
+  let estateForSale;
+  let snapshotId;
+  let provider;
+
+  before("", async () => {
+    ConfigLoader.setConfig(config);
+
+    provider = ProviderFactory.getProvider();
+
+    const network = await provider.getNetwork();
+    const { chainId } = network;
+    expect(chainId, "The target network isn't ropsten.").to.equal(
+      ROPSTEN_NETWORK_ID
+    );
+
+    manaContract = new Mana(MANA_ADDRESS);
+    landContract = new Land(LAND_ADDRESS);
+    estateContract = new Estate(ESTATE_ADDRESS);
+    marketplaceContract = new DecentralandMarketplace(MARKETPLACE_ADDRESS);
+
+    const decentralandUtils = new DecentralandUtils();
```

Comment:
Note - I wrote this question before reviewing the code for the `DecentralandUitls`, so while I understand what it does now, I'm still curious how you'd describe it's role nonetheless.

Reply:
The way that I've found to get data from `Marketplace` contract was using `etherscan` API for now (I think that we'll use subgraph sooner instead).
I've decided to create a separated class instead of creating functions inside `helpers.js` because such functions maybe will be useful and can be exposed by SDK for use from the app.
Since it's unstable and I'm not sure what is the best location for that class I've kept inside `testHelpers` dir.
Note: I'm not sure if it is the best name for the class and maybe will be better their functions are static.

Reply:
Makes sense. Let's move it outside of the `testHelpers` dir though, even if its location may change. Maybe just `helpers` for non-test helpers?

Yeah, it might end up being the case that the functions can be static. I haven't yet thought of why it would need to be a class (but it's very possible I'm missing something)

Reply:
Hmmm..
Are think that will be better if I move from `src/DecentralandUtils.js` (class) to `src/helpers/helpers.js` (just functions)?

Reply:
What do you think @pcowgill ?

Reply:
Hmm, I guess if there's any chance we'd want to add more functions to get things from that contract or using that provider, it's useful if that's already set up on an instance of the class.

I had to take another look at the class itself to decide, and I've done that now.

I still think moving it out of `testHelpers` into `helpers` would make sense. Or maybe a folder called "data fetching" or something like that.

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/218

Reply:
Thanks! Resolving